etna/transforms/timestamp/holiday.py

```diff
@@ -42,16 +74,17 @@ def __init__(self, iso_code: str = "RUS", mode: str = "binary", out_column: Opti
         iso_code:
             internationally recognised codes, designated to country for which we want to find the holidays
         mode:
-            `binary` to indicate holidays, `category` to specify which holiday do we have at each day
```

Comment:
You shouldn't delete `` symbols, also you have to add some info about `days_count` mode here.